packages/providers/upload-cloudinary/package.json

```diff
@@ -17,13 +17,13 @@
   },
   "license": "SEE LICENSE IN LICENSE",
   "author": {
-    "name": "Luca Perret",
-    "email": "perret.luca@gmail.com",
-    "url": "https://github.com/lucaperret"
+    "name": "Strapi Solutions SAS",
```

Comment:
you can keep the original author here

Reply:
Copy/paste gone wrong. Fixed it alongside a small adjustment in another provider to keep the same formatting.